### DISCUSSION TITLE
Question about the source codes of LAM-submodel

### DISCUSSION THREAD

**Discussion by Weisserlinks** [pybamm-team/PyBaMM#2050] [Q&A] [answered]
Dear Pybamm team, 

may I ask one question about the codes of the stress-driven LAM model:
` stress_h_surf = (stress_r_surf + 2 * stress_t_surf) / 3`
 ` stress_h_surf *= stress_h_surf > 0`
  `stress_h_surf_min = stress_h_surf * 0`
`j_stress_LAM = (
                -beta_LAM
                * ((stress_h_surf - stress_h_surf_min) / stress_critical) ** m_LAM
            )`
According to the literature **stress_h_surf** should be the maximal hydrostatic stress. How could it be calculated in this way? Why **stress_h_surf** multiplied itself?

Thanks a lot!

**Comment by brosaplanella** [comment]
In the [docs for the loss of active material model](https://pybamm.readthedocs.io/en/latest/source/models/submodels/active_material/loss_active_material.html) you can find the references where the models are defined, which should provide more details about why the stress is defined this way.

**Comment by valentinsulzer** [comment, marked as answer]
It is not multiplied by itself, it is multiplied by `(stress_h_surf > 0)` i.e. no stress if it's negative.
The maximum hydrostatic stress is hard to define numerically (leads to issues in the jacobian), which is why we assume it's the surface value
